document/src/test/java/com/yahoo/document/json/JsonReaderTestCase.java

```diff
@@ -719,15 +885,16 @@ public void testMapStringToArrayOfInt() throws IOException {
 
     @Test
     public void testOldMapStringToArrayOfInt() throws IOException {
-        JsonReader r = createReader(inputJson("{ 'put': 'id:unittest:testMapStringToArrayOfInt::whee',",
-                "  'fields': {",
-                "    'actualMapStringToArrayOfInt': [",
-                "      { 'key': 'bamse', 'value': [1, 2, 3] } ]}}"));
-        DocumentParseInfo parseInfo = r.parseDocument().get();
-        DocumentType docType = r.readDocumentType(parseInfo.documentId);
-        DocumentPut put = new DocumentPut(new Document(docType, parseInfo.documentId));
-        new VespaJsonDocumentReader(false).readPut(parseInfo.fieldsBuffer, put);
-        Document doc = put.getDocument();
+        Document doc = docFromJson("""
+                                   {
+                                     "put": "id:unittest:testMapStringToArrayOfInt::whee",
+                                     "fields": {
+                                       "actualMapStringToArrayOfInt": {
+                                         "bamse": [1, 2, 3]
```

Comment:
This previously appears to have tested a (presumed legacy) update syntax with explicit `"key": foo, "value": bar` mappings. With these changes it's identical to `testMapStringToArrayOfInt()`.

Consider either testing the explicit KV syntax (if we want to test it) or removing this test entirely (if we don't).

Reply:
Ah, my mistake. I tried to verify all the AI generated JSON, but evidently I failed here.